src/routes/recipe.js

```diff
@@ -32,25 +32,29 @@ router.get(
   ensurePermission('recipe', 'read'),
   [param('id').isNumeric().isInt({ min: 1 }).toInt()],
   handleValidationErrors(),
-  handleModelOperation(Recipe, 'findOne', (req) => ({
-    where: {
-      id: req.params.id
-    },
-    include: [
+  handleModelOperation(Recipe, 'findOne', (req) => {
```

Comment:
This could just be `handleModelOperation(Recipe, 'findOne', (req) => [{` with no return statement